srcpkgs/mesa/template

```diff
@@ -93,7 +93,7 @@ if [ "$_have_intel" ]; then
 	_have_vulkan=yes
 	_gallium_drivers+=",crocus,iris"
```

Comment:
```suggestion
	_gallium_drivers+=",crocus,iris,i915"
```
I suggest adding i915 to the gallium drivers list.  I cross-compiled this for i686 and tested on my EeePC.
`glxinfo -B` in X shows this output:
```name of display: :0
display: :0  screen: 0
direct rendering: Yes
Extended renderer info (GLX_MESA_query_renderer):
    Vendor: Mesa Project (0x8086)
    Device: i915 (chipset: 945GME) (0x27ae)
    Version: 22.0.2
    Accelerated: yes
    Video memory: 192MB
    Unified memory: yes
    Preferred profile: compat (0x2)
    Max core profile version: 0.0
    Max compat profile version: 2.1
    Max GLES1 profile version: 1.1
    Max GLES[23] profile version: 2.0
OpenGL vendor string: Mesa Project
OpenGL renderer string: i915 (chipset: 945GME)
OpenGL version string: 2.1 Mesa 22.0.2
OpenGL shading language version string: 1.20

OpenGL ES profile version string: OpenGL ES 2.0 Mesa 22.0.2
OpenGL ES profile shading language version string: OpenGL ES GLSL ES 1.0.16
```

I did notice that `sway` (wayland) uses the `softpipe` renderer.

Reply:
@zlice it looks like you lost the change that added i915 to the list of gallium drivers.